### PR TITLE
#167733634 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by danielufeli**
### What does this PR do?
Integrates TravisCI with readme badge

### Description of Task to be completed?
- Added and configured .travis.yml config file
- Added Travis badge to the readme file

### What are the relevant pivotal tracker stories?
#167733634

### Screenshots (if appropriate)
image

[Finishes #167733634]

### CODE REVIEW

**Review by dchima — CHANGES_REQUESTED** [review]
The travis CI is failing because you didn't add the following to the .travis.yml file:

**install:** 
     **- npm ci**
**script:**
    **- npm test**

The first will run travis in their site and the second will run whatever tests script we setup in the package.json. Maybe wait until the mocha test setup is approved before adding npm test portion

**Review by Baystef — APPROVED** [review]
Proof of Travis-ci spotted on the pull request and also badge is integrated properly

**Comment by Baystef** [comment]
> The travis CI is failing because you didn't add the following to the .travis.yml file:
> 
> **install:**
> **- npm ci**
> **script:**
> **- npm test**
> 
> The first will run travis in their site and the second will run whatever tests script we setup in the package.json. Maybe wait until the mocha test setup is approved before adding npm test portion

**Review by dchima — APPROVED** [review]
It seems Travis still fails until a test script is passed (it does automatic testing). 
So I'm approving this PR.

**Comment by efeguono** [comment]
I'm going to wait until the PR that adds unit tests to the project is merged so that this can be fully tested.

File: README.md
Comment (efeguono) [inline README.md:1]: Why was this removed?

File: .travis.yml
Comment (efeguono) [inline .travis.yml]: This requires a new line to be present at the end of the file

File: README.md
Comment (danielufeli) [reply README.md:1]: I made a PR to master before, but when I switched base to develop, it conflicted with Chima's code because he already merged to that branch, that's why I removed it.

File: .travis.yml
Comment (danielufeli) [reply .travis.yml]: ok @eguonoefe

File: README.md
Comment (efeguono) [reply README.md:1]: That is a very wrong way to resolve merge conflicts. If this was a new feature, It is definitely sure that you would be introducing a bug to the application.

Comment (efeguono) [reply README.md:1]: Always make enquiries (or be absolutely sure) before deleting any line of code. In any event, I have reverted the changes pertaining to the HoundCI integration. Be sure to rebase from develop and correctly resolve you merge conflicts

Comment (danielufeli) [reply README.md:1]: Ok thank you noted @eguonoefe

**Review by dchima — APPROVED** [review]
Travis is passing on initial build, and badge shows fine on readme. Maybe update the screenshot on the PR description to show passing.

**Comment by danielufeli** [comment]
> Travis is passing on initial build, and badge shows fine on readme. Maybe update the screenshot on the PR description to show passing.

ok updated screenshot

**Review by Segun-Ogundipe — APPROVED** [review]
This is great!!! I confirmed Travisci has been properly integrated

**Review by daylay92 — COMMENTED** [review]
A new line should be added at the end of every file including this one

File: .travis.yml
Comment (daylay92) [inline .travis.yml:8]: A new line should be added at the end of every file

Comment (danielufeli) [reply .travis.yml:8]: I added a new line. @daylay92

Comment (daylay92) [reply .travis.yml:8]: okay

Comment (daylay92) [reply .travis.yml:8]: Since the package.json file has been updated alongside the test script , I think you can now update the .travis.yml file with the install and script properties like so:
*install :* 
**npm install**
*script :*
**npm test**

**Pull request closed without merging**
